dali/base/dafdesc.cpp

```diff
@@ -3498,6 +3498,11 @@ bool GroupInformation::checkIsSubset(const GroupInformation & other)
 
 void GroupInformation::createStoragePlane(IPropertyTree * storage, unsigned copy) const
 {
+    // Check that storage plane does not already have a definition
+    VStringBuffer xpath("planes[@name='%s']", name.str());
+    if (storage->hasProp(xpath))
```

Comment:
In a future PR it might be interesting to allow some attributes for the implicit storage planes to be set in the configuration - so merge the definitions.  For example to be able to set expert options on thor storage planes, even if only for testing.

Reply:
I agree and have opened this [JIRA](https://hpccsystems.atlassian.net/browse/HPCC-33057).